lib/APIDigester/ModuleAnalyzerNodes.cpp

```diff
@@ -1601,8 +1601,9 @@ SwiftDeclCollector::constructTypeNode(Type T, TypeInitInfo Info) {
     // Still, return type first
     Root->addChild(constructTypeNode(Fun->getResult()));
 
-    auto Input = AnyFunctionType::composeTuple(Fun->getASTContext(),
-                                               Fun->getParams());
+    auto Input =
+        AnyFunctionType::composeTuple(Fun->getASTContext(), Fun->getParams(),
+                                      ParameterFlagHandling::IgnoreNonEmpty);
```

Comment:
@nkcsgexi This preserves the current behavior of ignoring the parameter flags, but might we want to record parameter flags here for e.g `inout`, variadics, or ownership? AFAIK those would affect ABI. Should I file a bug?

Reply:
Thanks for pinging, @hamishknight ! Wouldn't these flags be reflected in the mangled names of the function? We diagnose mangled name changes as ABI breakage.

Reply:
Ah yes, in which case I guess this is fine